addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-windows-instance.cfn.yml

```diff
@@ -333,7 +333,8 @@ Resources:
           # Download s3-synchronizer Windows executable
           $uri = [System.Uri]"${EnvironmentInstanceFiles}"
           $key = "$($uri.AbsolutePath.Substring(1))/bin/s3-synchronizer-windows-amd64.exe"
-          Read-S3Object -BucketName $uri.Host -Key $key -File s3-synchronizer.exe
+          $region = "${AWS::Region}"
```

Comment:
still curious why it works depending on setting a shell variable without using it

Reply:
I think I misunderstand your meaning,  Yes, $region = "${AWS::Region}" should not be needed,  because I  use "${AWS::Region}" directly.
I will remove it.